services/QuillLMS/db/migrate/20241025214425_add_feedback_history_indices.rb

```diff
@@ -0,0 +1,27 @@
+class AddFeedbackHistoryIndices < ActiveRecord::Migration[7.1]
+  def up
+
+    execute <<~SQL
+      CREATE INDEX index_feedback_histories_on_rule_uid_and_prompt_id_and_activity_version_and_used_include_id_and_metadata
+      ON feedback_histories(rule_uid, prompt_id, activity_version, used)
+      INCLUDE (id, metadata);
```

Comment:
AREL does not support the postgres INCLUDE directive, so raw SQL is required

Reply:
The `add_index` lists an include parameter: h
ttps://api.rubyonrails.org/v7.1/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_index, as well as `algorithm` for concurrently.

```
  def up
    add_index :feedback_histories,
      [:rule_uid, :prompt_id, :activity_version, :used],
      name: 'index_feedback_histories_on_multi_columns',
      algorithm: :concurrently,
      include: [:id, :metadata]
  end
```